writeengine/server/we_dmlcommandproc.cpp

```diff
@@ -70,10 +70,8 @@ WE_DMLCommandProc::WE_DMLCommandProc()
     if (fpc.length() != 0)
         filesPerColumnPartition = cf->uFromText(fpc);
 
-    string epsf = cf->getConfig("ExtentMap", "ExtentsPerSegmentFile");
-
-    if (epsf.length() != 0)
-        extentsPerSegmentFile = cf->uFromText(epsf);
+    // MCOL-4685: remove the option to set more than 2 extents per file (ExtentsPreSegmentFile).
+    extentsPerSegmentFile = 2;
```

Comment:
We should use the constant defined in dbcon/joblist/columncommand-jl.h if this doesn't make a circular dependency.